src/Microsoft.PowerApps.TestEngine.Tests/TestInfra/PlaywrightTestInfraFunctionTests.cs

```diff
@@ -147,6 +147,8 @@ public async Task SetupAsyncTest(string browser, string device, int? screenWidth
         [Theory]
         [InlineData("")]
         [InlineData(null)]
+        [InlineData("Chrome")]
+        [InlineData("Safari")]
```

Comment:
To demonstrate to whomever encounters this test later on, might want to use an obviously invalid browser name. Something like "INVALID BROWSER NAME" to make it clear that we wouldn't expect to find a browser with this name on the test machine (and also to not conflict with a name that might actually be valid on someone's computer).

Reply:
Sure, I can add one to this.